assets/style.css

```diff
@@ -43,6 +43,7 @@ html,
 body {
     margin: 0;
     border: 0;
+    height: 100%;
```

Comment:
Could you make this `min-height` and add a selector for HTML with a height of `100%`?